pkg/gitproviders/provider.go

```diff
@@ -498,6 +500,10 @@ func (p defaultGitProvider) GetCommitsFromOrgRepo(orgRepRef gitprovider.OrgRepos
 	return commits, nil
 }
 
+func (p defaultGitProvider) GetProviderName() GitProviderName {
+	return providerName
```

Comment:
:warning: Global State alarm bells :warning:

What happens when two concurrent HTTP requests try to set this variable at the same time?